pkg/registry/core/serviceaccount/registry.go

```diff
@@ -67,12 +67,12 @@ func (s *storage) GetServiceAccount(ctx genericapirequest.Context, name string,
 }
 
 func (s *storage) CreateServiceAccount(ctx genericapirequest.Context, serviceAccount *api.ServiceAccount) error {
```

Comment:
only safe due to usage.  This one should be removed in favor of loopback too.

Reply:
Also dead code fixed via #54693